logtail/logtail.go

```diff
@@ -220,6 +222,11 @@ func (l *Logger) SetLinkMonitor(lm *monitor.Mon) {
 	l.linkMonitor = lm
 }
 
+// SetSockstatsLabel sets the label used in sockstat logs to identify network traffic from this logger.
```

Comment:
I originally put it in the Config struct, but the zero value of `sockstats.Label` is meaningful, so then I had to make it a pointer, and then you can't take the address of a constant for the default value, and then I just stopped fighting it and added the method.  🤷🏻 

And no, it's not currently being called.  I was sure if you were suggesting that we should go ahead and add a new label for sockstat logs now, or just have this available as a debugging tool in the future.

Reply:
If you don't mind bumping `sockstats.Label` so that it has a real zero value, that would certainly make this easier.  I just wasn't sure how you felt about changing that at this point.

Reply:
Renumbering seems awkward given that there already are clients that are uploading metrics with this -- sorry for not leaving 0 from the initial set of values.

I see what you mean about the pointer value not working  (would it be too gross to wrap it in an array?), so this function seems fine.

I still think we should use it in `sockstatlog` (and `netlog`) now, as opposed to wishing we had it when debugging in the future.